src/layman/layer/geoserver/geoserver_test.py

```diff
@@ -25,3 +30,77 @@ def test_check_user_wms():
     assert exc_info.value.http_code == 400
     assert exc_info.value.code == 45
     assert exc_info.value.data['workspace_name'] == user
+
+
+def assert_wfs_bbox(workspace, layer, expected_bbox):
+    wfs_layer = f"{workspace}:{layer}"
+    wfs_get_capabilities = wfs.get_wfs_proxy(workspace)
+    wfs_bbox_4326 = wfs_get_capabilities.contents[wfs_layer].boundingBoxWGS84
+    wfs_bbox_3857 = bbox_util.transform(wfs_bbox_4326, 4326, 3857, )
+    test_util.assert_same_bboxes(expected_bbox, wfs_bbox_3857, 0.00001)
+
+
+def assert_wms_bbox(workspace, layer, expected_bbox):
+    wms_get_capabilities = wms.get_wms_proxy(workspace)
+    wms_bboxes = wms_get_capabilities.contents[layer].crs_list
+    for bbox in wms_bboxes:
```

Comment:
`wms_bbox_3857 = next(bbox[:4] for bbox in wms_bboxes if bbox[4] == 'EPSG:3857')`